### PR TITLE
Add hyperterm

### PR DESCRIPTION
**Pull request by Dremora**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

File: Casks/hyperterm.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask 'hyperterm' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://hyperterm-updates.now.sh/download/osx'
```
Comment (vitorgalvao) [inline Casks/hyperterm.rb]: Use versioned link from [Github releases](https://github.com/zeit/hyperterm/releases) instead, and add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

You’ll also need to add a `# github.com/zeit/hyperterm was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/hyperterm.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+cask 'hyperterm' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/hyperterm.rb]: This will be versioned after you use the Github releases.

**Comment by Dremora** [comment]
All done. Also added `~/.hyperterm.js` and `~/.hyperterm_plugins` to the `zap` stanza.

File: Casks/hyperterm.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+cask 'hyperterm' do
+  version '0.4.5'
+  sha256 '06b9ebf6dbc8c3cc277b3fede2a7945778edce950586bbd87745d5d9a370601a'
+
+  # github.com/zeit/hyperterm was verified as official when first introduced to the cask
+  url 'https://github.com/zeit/hyperterm/releases/download/0.4.5/hyperterm-macos-x64-0.4.5.zip'
```
Comment (vitorgalvao) [inline Casks/hyperterm.rb]: `url "https://github.com/zeit/hyperterm/releases/download/#{version}/hyperterm-macos-x64-#{version}.zip"`

Comment (Dremora) [inline Casks/hyperterm.rb]: Oops. Fixed.

**Comment by vitorgalvao** [comment]
Thank you for the fast turnaround. Merging when Travis passes.

**Comment by Dremora** [comment]
It seems `uninstall` is failing when HyperTerm is running:

```
$ brew cask uninstall hyperterm     
==> Removing App: '/Applications/HyperTerm.app'
==> Running uninstall process for hyperterm; your password may be necessary
==> Quitting application ID co.zeit.hyperterm
Password:
==> 43:47: execution error: HyperTerm got an error: User canceled. (-128)
Error: Command failed to execute!

==> Failed command:
["/usr/bin/sudo", "-E", "--", "/usr/bin/osascript", "-e", "tell application id \\\"co.zeit.hyperterm\\\" to quit"]

==> Output of failed command:


==> Exit status of failed command:
#<Process::Status: pid 28707 exit 1>
```

Not sure what's causing the issue.

**Comment by vitorgalvao** [comment]
> Shall I just remove this stanza for now

Yes, we have an issue about automating that quitting anyway.

**Comment by Dremora** [comment]
Done.

**Pull request merged**